swift/apple/FirezoneKit/Sources/FirezoneKit/Views/ResourceMenuSection.swift

```diff
@@ -0,0 +1,277 @@
+//
+//  ResourceMenuSection.swift
+//  (c) 2025 Firezone, Inc.
+//  LICENSE: Apache-2.0
+//
+
+import SwiftUI
+
+/// Individual resource menu item with submenu containing resource details
+struct ResourceMenuItem: View {
+  let resource: Resource
+  @EnvironmentObject var store: Store
+
+  var body: some View {
+    Menu {
+      ResourceDetailsSubmenu(resource: resource)
+    } label: {
+      HStack(spacing: 4) {
+        if let symbolName = resourceSymbol {
+          Image(systemName: symbolName)
+        }
+        Text(resource.name)
+      }
+    }
+  }
+
+  var resourceSymbol: String? {
+    if resource.isInternetResource() {
+      return store.configuration.internetResourceEnabled ? "network" : "network.slash"
```

Comment:
Do other clients' icons need to be updated as well to maintain the current consistency across platforms?